Engine/ScriptAnalyzer.cs

```diff
@@ -1684,22 +1698,21 @@ private static Range SnapToEdges(EditableText text, Range range)
         }
 
         private static IEnumerable<CorrectionExtent> GetCorrectionExtentsForFix(
-            IEnumerable<CorrectionExtent> correctionExtents)
+            List<CorrectionExtent> correctionExtents)
```

Comment:
If one goes up in the call tree of the `Fix` methods then one sees that the passed through `correctionExtents` object is already of type `List` [here](https://github.com/PowerShell/PSScriptAnalyzer/blob/bd5a4b1f9867bdad301b7e788343c4ce29fd8bf5/Engine/ScriptAnalyzer.cs#L1599), therefore this allows us to drop calls to `.ToList()` and `.Count()` here.